Controller/MediaAdminController.php

```diff
@@ -77,10 +77,11 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context); 
```

Comment:
I think the failure comes from to long lines. try this:

$categoryManager = $this->container->get('sonata.classification.manager.category');
$rootCategory = $categoryManager->getRootCategory($context); 

...

$categoryByContext = $categoryManager->findOneBy(ar...